kqueen_ui/blueprints/ui/templates/ui/partial/footer.html

```diff
@@ -1,9 +1,11 @@
   <footer>
+    {# Uncomment following lines for demo-releases
```

Comment:
I will change the footer and remove Beta stripe after 1.0 release. Keep it in for now

Reply:
ok